File: src/main/java/org/example/kalkulationsprogramm/repository/ZahlungsartRepository.java
package org.example.kalkulationsprogramm.repository;

import org.example.kalkulationsprogramm.domain.Zahlungsart;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ZahlungsartRepository extends JpaRepository<Zahlungsart, Long> {
    List<Zahlungsart> findAllByOrderBySortierungAscBezeichnungAsc();
    List<Zahlungsart> findByAktivTrueOrderBySortierungAscBezeichnungAsc();
}

File: src/main/kotlin/org/example/kalkulationsprogramm/repository/ZahlungsartRepository.kt
package org.example.kalkulationsprogramm.repository

import org.example.kalkulationsprogramm.domain.Zahlungsart
import org.springframework.data.jpa.repository.JpaRepository

interface ZahlungsartRepository : JpaRepository<Zahlungsart, Long> {
    fun findAllByOrderBySortierungAscBezeichnungAsc(): List<Zahlungsart>

    fun findByAktivTrueOrderBySortierungAscBezeichnungAsc(): List<Zahlungsart>
}
